src/lib/Microsoft.Health.Common/Extension/ExceptionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Common.Extension
+{
+    public static class ExceptionExtensions
+    {
+        private const string LogToCustomerAttribute = "LogToCustomer";
```

Comment:
Per our conversation yesterday, the concern is the concept of customer is very PaaS centric and in the OSS it doesn't have  a direct application.  

What do you think of naming this something like FowardLog or LogFowarding to make it a little more generic and OSS appropriate?

Reply:
Another option is we look at moving this into PaaS but update OSS with an exception factory concept and have a default implementation.  In PaaS we can inject another one, and as part of the factories logic it would set this extension when creating them.

Reply:
I am a bit confused on how we would follow the factory approach. What do you think the API for tagging arbitrary logs would be?

Reply:
Sure, I will try will look at adding an example